lib/sprinkler_web/tortoise/mqtt_handler.ex

```diff
@@ -0,0 +1,42 @@
+defmodule SprinklerWeb.Tortoise.MqttHandler do
```

Comment:
I think the namespace should be Mqtt
```suggestion
defmodule SprinklerWeb.Mqtt.Handler do
```